app/views/links/history.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="left">
```

Comment:
We already have a template for a list of PastLink records, do you think we could use that?

[app/views/users/_past_links.html.erb](https://github.com/pupgray/walltaker/blob/main/app/views/users/_past_links.html.erb)

Can be edited to show the setter name as well, but I prefer this grid layout.

I may also drop the link preview, since it doesn't add too much here.

Reply:
You can drop the link preview, not a big deal. I put it there because the page just felt kinda empty without it.

As for that partial: The main reason it was done this way was to give posts equal spacing, along with some additional breathing room in case I wanted to add new buttons/icons. I wanted the equal spacing so that usernames don't get cut off based on the aspect ratio of the image.